lib/profile/view_profile/sections/skills.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import '../../../constants/profile_constants.dart';
+import '../../common/section.dart';
+import '../../common/skill_row.dart';
+ 
+class ProfileSkills extends StatelessWidget {  
+  final List<Map<String, Object>> skills;
+  ProfileSkills({Key key, @required this.skills}) : super(key: key);
+  
+  @override
+  Widget build(BuildContext context) {
+    return Section(
+      title: sectionTitleSkills,
+      child: buildSkillList(context)
+    );
+  }
+
+  Widget buildSkillList(BuildContext context) {
+    List<Widget> expertList = <Widget>[];
+    List<Widget> intermediateList = <Widget>[];
+    List<Widget> beginnerList = <Widget>[];
+
+    // Temporary workaround for sorting skills
```

Comment:
Remove

Reply:
Same reply to my comment above on temporary workarounds!